lnbits/core/crud.py

```diff
@@ -395,21 +395,54 @@ async def get_payments(
         clause.append("checking_id NOT LIKE 'temp_%'")
         clause.append("checking_id NOT LIKE 'internal_%'")
 
+    return await (conn or db).fetch_page(
```

Comment:
- where is `fetch_page` defined?
   - I can find for example `fetchall` in `sqlalchemy_aio.base.AsyncResultProxy`
- also please note the `rewrite_query` calls from `lnbits/db.py/Connection`

Reply:
its defined int db.py aswell, its a fetch_page is a method of Database class.